source/Calamari.Common/Features/FunctionScriptContributions/FunctionAppenderScriptWrapper.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Calamari.Common.Features.Processes;
+using Calamari.Common.Features.Scripting;
+using Calamari.Common.Features.Scripts;
+using Calamari.Common.Plumbing.Extensions;
+using Calamari.Common.Plumbing.FileSystem;
+using Calamari.Common.Plumbing.Variables;
+using Newtonsoft.Json;
+
+namespace Calamari.Common.Features.FunctionScriptContributions
+{
+    class FunctionAppenderScriptWrapper: IScriptWrapper
+    {
+        readonly IVariables variables;
+        readonly ICalamariFileSystem fileSystem;
+        readonly CodeGenFunctionsRegistry codeGenFunctionsRegistry;
+
+        public FunctionAppenderScriptWrapper(IVariables variables, ICalamariFileSystem fileSystem, CodeGenFunctionsRegistry codeGenFunctionsRegistry)
+        {
+            this.variables = variables;
+            this.fileSystem = fileSystem;
+            this.codeGenFunctionsRegistry = codeGenFunctionsRegistry;
+        }
+
+        public int Priority { get; } = ScriptWrapperPriorities.ToolConfigPriority;
+        public IScriptWrapper NextWrapper { get; set; }
+
+        public bool IsEnabled(ScriptSyntax syntax)
+        {
+            if (String.IsNullOrEmpty(variables.Get("Octopus.Sashimi.ScriptFunctions.Registration")))
```

Comment:
This literal should be a constant somewhere, same applies to the Sashimi/Server repo's